### PR TITLE
form 526 upload BDD instructions

### PR DESCRIPTION
**Pull request by annaswims**
## Description of change
Upload a document with instructions so that claims processors are reminded the rules of BDD. 

## Original issue(s)
department-of-veterans-affairs/va.gov-team#18868

### CODE REVIEW

File: app/workers/evss/disability_compensation_form/upload_bdd_instructions.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module EVSS
+  module DisabilityCompensationForm
+    class UploadBddInstructions < Job
+      FORM_TYPE = '21-526EZ'
+      STATSD_KEY_PREFIX = 'worker.evss.submit_form526_bdd_instructions'
```
Comment (LindseySaari) [inline app/workers/evss/disability_compensation_form/upload_bdd_instructions.rb:7]: Are the constants on lines 6 and 7 used at all?

Comment (annaswims) [reply app/workers/evss/disability_compensation_form/upload_bdd_instructions.rb:6]: STATSD_KEY_PREFIX is referenced in lib/common/client/concerns/monitoring.rb:
   22:           increment("#{self.class::STATSD_KEY_PREFIX}.#{caller}.total")
   30:           increment("#{self.class::STATSD_KEY_PREFIX}.#{caller}.fail", tags)

**Review by LindseySaari — APPROVED** [review]

File: app/workers/evss/disability_compensation_form/upload_bdd_instructions.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      sidekiq_options retry: 14
+
+      # Submits a BDD instruction PDF in to EVSS
+      #
+      # @param submission_id [Integer] The {Form526Submission} id
+      #
+      def perform(submission_id)
+        super(submission_id)
+        with_tracking('Form526 Upload BDD instructions:', submission.saved_claim_id, submission.id) do
+          file_body = File.read('lib/evss/disability_compensation_form/bdd_instructions.pdf')
```
Comment (edmangimelli) [inline app/workers/evss/disability_compensation_form/upload_bdd_instructions.rb:18]: I'm a fan of [this style](https://github.com/department-of-veterans-affairs/vets-api/blob/8546477896cc1247aa5d2d54866007df13eb5bec/spec/factories/va0994.rb#L6
) of reading a file, but I see this relative, forward-slash-hard-coded style all over vets-api --so it doesn't look like there's a consensus

**Review by edmangimelli — APPROVED** [review]
🚀

**Pull request merged**